src/System.Private.Uri/src/System/Uri.cs

```diff
@@ -2109,12 +2109,18 @@ private unsafe ParsingError PrivateParseMinimal()
                             _flags |= Flags.UncPath;
                             idx = i;
                         }
+                        else if (!IsWindowsSystem && _syntax.InFact(UriSyntaxFlags.FileLikeUri) && i - idx == 3 && pUriString[i - 1] == '/')
+                        {
```

Comment:
These are left unchanged, only the file:/// case is detected.
What are your suggestions for these urls? Leave as is? Or change?
Should the authority 'localhost' be handled differently? Or any authority?
`file://foo%5Cbar` has an authority of `foo\bar` and is an invalid url (file without a path component)? Or Windows style: authority of 'foo' and path '/bar'?